bench/wtperf/wtperf.c

```diff
@@ -1759,41 +1468,6 @@ execute_populate(WTPERF *wtperf)
       "load ops/sec: %" PRIu64,
       print_secs, print_ops_sec);
 
-    /*
-     * If configured, compact to allow LSM merging to complete. We set an unlimited timeout because
-     * if we close the connection then any in-progress compact/merge is aborted.
-     */
-    if (opts->compact) {
-        assert(opts->async_threads > 0);
```

Comment:
What's the relationship between `compact` and `async_threads`?  These lines of code seem to imply that if `compact` is true, then we must have `async_threads`.  But since we're getting rid of async, why aren't we getting rid of `compact`?

Reply:
I will have to look into that.